spec/index.bs

```diff
@@ -3503,10 +3500,6 @@ truly optional.
         Indicates the required {{GPUTextureViewDescriptor/dimension}} for texture views bound to
         this binding.
 
-        Note:
-            This enables Metal-based WebGPU implementations to back the respective bind groups with
-            `MTLArgumentBuffer` objects that are more efficient to bind at run-time.
-
```

Comment:
Please leave a link back to https://github.com/gpuweb/gpuweb/pull/339 in a `<!-- comment -->` or `Note:`